week3/predict_citibike.Rmd

```diff
@@ -0,0 +1,264 @@
+---
+title: "Predict Citibike trips"
+output: html_document
+date: '2022-06-16'
+---
+
+```{r}
+library(here)
+library(scales)
+library(tidyverse)
+library(modelr)
+library(lubridate)
+library(broom)
+
+theme_set(theme_bw())
+options(repr.plot.width=4, repr.plot.height=3)
+
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+1. Trips per day file
+
+```{r}
+
+trips_per_day <- read_tsv('trips_per_day.tsv') %>%
+  mutate(day = wday(ymd), weekday = (day < 6))
+
+holiday <- read_csv('holidays.csv', col_names = FALSE)

```

Comment:
If you include outside data you should include it, or link to it so that people can run your code easily. If that's not an option, you can use the knit feature of RStudio to make a copy of your exploration that people can read without running your code.